corehq/apps/accounting/tasks.py

```diff
@@ -616,3 +618,102 @@ def assign_explicit_community_subscription(domain_name, start_date):
+    from corehq.apps.domain.views import DomainBillingStatementsView
+
+    today = today or datetime.date.today()
+    invoices = Invoice.objects.filter(is_hidden=False,
+                                      subscription__service_type=SubscriptionType.PRODUCT,
+                                      date_paid__isnull=True,
+                                      date_due__lt=today)\
+        .exclude(subscription__plan_version__plan__edition=SoftwarePlanEdition.ENTERPRISE)\
+        .order_by('date_due')\
+        .select_related('subscription__subscriber')
+
+    domains = set()
+    for invoice in invoices:
+        if invoice.get_domain() not in domains:
+            domains.add(invoice.get_domain())
+            total = Invoice.objects.filter(is_hidden=False,
+                                           subscription__subscriber__domain=invoice.get_domain())\
+                .aggregate(Sum('balance'))['balance__sum']
+            if total >= 100:
+                domain = Domain.get_by_name(invoice.get_domain())
+                current_subscription = Subscription.get_subscribed_plan_by_domain(domain)[1]
+                if not current_subscription.skip_auto_downgrade:
+                    days_ago = (today - invoice.date_due).days
+                    context = {
+                        'domain': invoice.get_domain(),
+                        'total': total,
+                        'subscription_url': absolute_reverse(DomainSubscriptionView.urlname,
+                                                             args=[invoice.get_domain()]),
+                        'statements_url': absolute_reverse(DomainBillingStatementsView.urlname,
+                                                           args=[invoice.get_domain()]),
+                        'date_60': invoice.date_due + datetime.timedelta(days=60),
+                        'contact_email': settings.INVOICING_CONTACT_EMAIL
+                    }
+                    if days_ago == 61:
+                        _downgrade_domain(current_subscription)
+                        _send_downgrade_notice(invoice, context)
+                    elif days_ago == 58:
+                        _send_downgrade_warning(invoice, context)
+                    elif days_ago == 30:
+                        _send_overdue_notice(invoice, context)
+                    elif days_ago == 1:
+                        _create_overdue_notification(invoice, context)
+
+
+def _send_downgrade_notice(invoice, context):
+    send_html_email_async.delay(
+        ugettext('Oh no! Your CommCare subscription for {} has been downgraded'.format(invoice.get_domain())),
+        invoice.contact_emails,
+        render_to_string('accounting/downgrade.html', context),
+        render_to_string('accounting/downgrade.txt', context),
+        cc=[settings.ACCOUNTS_EMAIL],
+        email_from=get_dimagi_from_email()
+    )
+
+
+def _downgrade_domain(subscription):
+    subscription.change_plan(
```

Comment:
I think you want to pass ```internal_change=True``` to ```change_plan``` so the  correct version of accounts@ gets the email: https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/accounting/emails.py#L36